assets/js/vue/utils/template-switch.js

```diff
@@ -0,0 +1,60 @@
+const grpLabels = {
+    status: 'Dataset Status',
+    fundingCycle: 'Grant Awards',
+    researchGroup:  'Projects',
+    fundingOrg: 'Funding Organizations',
+};
+const grpFlags = {
+    showFundingCycles: false,
+    showProjectDirector: true
+};
+const griidcLabels = {
+    status: 'Dataset Status',
+    fundingCycle: 'Funding Cycles',
+    researchGroup:  'Research Groups',
+    fundingOrg: 'Funding Organizations',
+};
+const griidcFlags = {
+    showFundingCycles: true,
+    showProjectDirector: false
+};
+
+export default {
+    getLabel: (name) => {
```

Comment:
Do we need these functions which all do the same thing?, Could we just have one function which checks if it is GRIIDC or GRP and return the variable value/text based on it